e2e/tests/router-navigation.spec.ts

```
@@ -76,11 +79,19 @@ test.describe("session navigation via URL", () => {
       .click();
     await page.waitForURL(new RegExp(SESSION_A.id));
 
+    await openHistoryWithSessions(page);
+    await page
+      .locator(`[data-testid="session-item-${SESSION_B.id}"]`)
+      .click();
```

ESLint

[error] Replace `⏎······.locator(`[data-testid="session-item-${SESSION_B.id}"]`)⏎······` with `.locator(`[data-testid="session-item-${SESSION_B.id}"]`)`

(prettier/prettier)